templates/terraform/resource.erb

```diff
@@ -299,7 +344,11 @@ func resource<%= resource_name -%>Delete(d *schema.ResourceData, meta interface{
 	return nil
 }
 
+
```

Comment:
probably don't need this extra newline, I think it'll get gofmtted away anyway

Reply:
That was an accident, thanks for the catch.